webpush/push.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	gcmURL     = "https://android.googleapis.com/gcm/send"
-	tempGcmURL = "https://gcm-http.googleapis.com/gcm"
+	tempGcmURL = "https://fcm-http.googleapis.com/fcm/send"
```

Comment:
what if someone still has the old, gcm-http.googleapis.com/gcm, stored somewhere in their records?